lib/FIR/afforestation.cc

```diff
@@ -25,6 +25,10 @@
 
 namespace Fortran::FIR {
 namespace {
+template<typename A> Expression *ExprRef(const A &a) {
```

Comment:
It looks like `A` has to be `parser::Expr`. Nothing else in the parse tree has `typedExpr`. So this doesn't need to be a template.

Reply:
ok